mmif/serialize/mmif.py

```diff
@@ -511,7 +511,75 @@ def get_view_contains(self, at_types: Union[ThingTypesBase, str, List[Union[str,
                 if at_types in view.metadata.contains:
                     return view
         return None
-    
+
+    def _is_in_time_between(self, start: Union[int, float], end: Union[int, float], annotation: Annotation) -> bool:
+        s, e = self.get_start(annotation), self.get_end(annotation)
+        return (s < start < e) or (s > start and e < end) or (s < end < e)
+
+    def _handle_time_unit(self, input_unit: str, ann_unit: str,
```

Comment:
this might be helpful. 

https://github.com/clamsproject/mmif-python/blob/c42234ff934bf758617001db6a3fff69c6bc8a87/mmif/utils/timeunit_helper.py#L41